source/indexes/create-vector-search-index.txt

```diff
@@ -71,7 +71,7 @@ Steps
          * - ``numDimensions``
            - int 
            - The number of vector dimensions, which Atlas Search enforces at index- and 
-             query-time. This value can't be greater than 2048.
+             query-time. This value can't be greater than 4096.
         
```

Comment:
I think it would be useful to add `type` also to this list as that is also an accepted field. In the Atlas UI, user needn't specify type, but that is required here. 
```suggestion
  * - ``type``
     - string
     - Human-readable label that identifies the type of index. Value must 
       be ``vector`` to perform vector search against the indexed fields. If omitted, 
       defaults to ``search``, which only supports full-text search.
```

Reply:
Thank you, updated!